### DISCUSSION TITLE
Is there a way to reorder the jobs via the API?

### DISCUSSION THREAD

**Discussion by jburd0** [smartin015/continuousprint#227] [General]
Just like the title, I would like to add a job via the API and move it to the top of the queue.

**Comment by smartin015** [comment]
Hey @jburd0 - there's not currently a public API method, however I plan to publish one in a coming release. 

If you'd rather not wait you can use the private [/job/mv](https://github.com/smartin015/continuousprint/blob/master/continuousprint/api.py#L207) method, but it may change without notice on future releases.
